static/js/enketo/widgets/android-datepicker.js

```diff
@@ -60,7 +60,7 @@ define( function( require, exports, module ) {
     Androiddatepicker.prototype.constructor = Androiddatepicker;
 
     Androiddatepicker.prototype._init = function() {
-        if ( !window.medicmobile_android || !window.medicmobile_android.datePicker ) {
+        if ( !window.medicmobile_android || typeof indow.medicmobile_android.datePicker !== 'function' ) {
```

Comment:
This typo is failing the build. jshint got one!!

Reply:
:¬) I made the same typo on the previous line too (copy/pasta?), but fixed it, so I thought I'd fixed the build

Reply:
Oh no, fixed but didn't push!